### DISCUSSION TITLE
Recovering Organic Maps bookmarks from a damaged phone

### DISCUSSION THREAD

**Discussion by HimmyWolf** [organicmaps/organicmaps#8315] [Q&A]
Hello!

I came across this forum through a google search about the location of the Organic Maps boomarks data. I first replied to an old thread, but as this issue is a little bit different I thought I'd make a new one instead.

The problem is that I've possibly lost my android phone to water damage. It will become more clear tomorrow at a phone repair shop. It may come down to data recovery from the motherboard or other sort of data recovery, if the phone can't be fixed. For recovering photos and other auch files it won't be an issue, but I am worried about the big list of bookmarks that I had saved on my travela through Organic Maps. The last KMZ file export and back up that I did was several months ago so a lot of important bookmarks would be lost. Is there any way to access those bookmarks through data recovery - using a laptop for example to view the recovered files or for example copying them to an SD card which would be inserted into a smart phone that has the same model and same Android version? Or is there no other way to salvage those bookmarks other than to recover the phone's function and open the application itself?

Thank you in advance
Kaspar

**Comment by biodranik** [comment]
Bookmarks are now stored in the system mount point at `/data/data/app.organicmaps/files/`, it can be accessed in theory, but it may not be an easy task.

If you had Google Backup enabled in system settings, you may try to restore another phone from Google Backup, and bookmarks should be restored.

There are several ideas to improve the situation, like online backup and sync #2082 and some kind of automated local export (which is complicated starting from Android 11)
